src/main/java/org/swisspush/redisques/util/DefaultRedisquesConfigurationProvider.java

```diff
@@ -46,19 +48,18 @@ public Result<Void, String> updateConfiguration(JsonObject configuration, boolea
     }
 
     private Result<Void, String> setConfigurationValues(JsonObject configurationValues, boolean validateOnly) {
-        if (configurationValues != null) {
+        if (configurationValues != null && !configurationValues.isEmpty()) {
             List<String> notAllowedConfigurationValues = findNotAllowedConfigurationValues(configurationValues.fieldNames());
             if (notAllowedConfigurationValues.isEmpty()) {
                 try {
                     Long processorDelayMaxValue = configurationValues.getLong(PROCESSOR_DELAY_MAX);
-                    if (processorDelayMaxValue == null) {
-                        return Result.err("Value for configuration property '" + PROCESSOR_DELAY_MAX + "' is missing");
-                    }
+                    Long processorTimeoutValue = configurationValues.getLong(PROCESSOR_TIMEOUT);
+
```

Comment:
```patch
-  if (processorDelayMaxValue == null) {
-      return Result.err("Value for configuration property '" + PROCESSOR_DELAY_MAX + "' is missing");
-  }
```
I assume it is by intent to remove this check?

Reply:
Yes it is by intent